app/gui/src/controller/searcher.rs

```diff
@@ -394,6 +394,8 @@ pub struct Filter {
     pub pattern: ImString,
     /// Additional context. A string representation of the edited accessor chain.
     pub context: Option<ImString>,
+    /// The name of the currently active module.
+    module_name: Rc<QualifiedName>,
```

Comment:
Please add a short info why `module_name` affects the filtering, perhaps with reference to `update_matching_info` or `code_with_static_this`.